CHANGES.txt

```diff
@@ -99,6 +99,8 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
       describe the Microsoft C++ compiler. Update the version table slightly.
       Amplified the usage of MSVC_VERSION.
     - Improve SharedLibrary docs a bit.
+    - Update warnings module: adds docstrings, drop three unused warnings,
```

Comment:
Please add the list of warnings being removed in the release notes and changes.txt

Reply:
Added those, and the two that now appear in the manpage.